pkg/controllers/user/rbac/project_handler.go

```diff
@@ -114,15 +150,26 @@ func (p *pLifecycle) ensureDefaultNamespaceAssigned(project *v3.Project) error {
 		}
 		ns.Annotations[projectIDAnnotation] = fmt.Sprintf("%v:%v", p.m.clusterName, project.Name)
 		if _, err := p.m.workload.Core.Namespaces(p.m.clusterName).Update(ns); err != nil {
-			return nil, err
-		}
-
-		return nil, nil
-	})
-	if updateCluster {
-		if _, err := p.m.workload.Management.Management.Clusters("").ObjectClient().Update(cluster.Name, c); err != nil {
 			return err
 		}
 	}
-	return err
+	return nil
+}
+
+func GetDefaultProjectsToNamespaces() (map[string][]string, error) {
```

Comment:
may be make it lowercase

Reply:
after making it lowercase, saw that `namespace_handler` uses this function for similar logic - assigning namespaces to default and system project during startup

Reply:
but theyre in the same package, arent they?